optaplanner-examples/src/test/resources/org/optaplanner/examples/lowentitycount/nearbySelection.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<solver>
+  <environmentMode>PRODUCTION</environmentMode>
+  <solutionClass>org.optaplanner.examples.tsp.domain.TspSolution</solutionClass>
+  <entityClass>org.optaplanner.examples.tsp.domain.Visit</entityClass>
+  
+  <scoreDirectorFactory>
+    <incrementalScoreCalculatorClass>org.optaplanner.examples.tsp.solver.score.TspIncrementalScoreCalculator</incrementalScoreCalculatorClass>
+    <initializingScoreTrend>ONLY_DOWN</initializingScoreTrend>
+  </scoreDirectorFactory>
+ 
+ <constructionHeuristic>
+    <constructionHeuristicType>FIRST_FIT_DECREASING</constructionHeuristicType>
+  </constructionHeuristic>
+
+  <termination>
+    <secondsSpentLimit>5</secondsSpentLimit>
+  </termination>
+
+  <localSearch>
+    <termination>
+      <stepCountLimit>1</stepCountLimit>
+    </termination>
+      <unionMoveSelector>
+          <changeMoveSelector>
+              <entitySelector id="entitySelector1"/>
+              <valueSelector>
+                  <nearbySelection>
+                      <originEntitySelector mimicSelectorRef="entitySelector1"/>
+                      <nearbyDistanceMeterClass>org.optaplanner.examples.tsp.domain.solver.nearby.VisitNearbyDistanceMeter</nearbyDistanceMeterClass>
+                      <parabolicDistributionSizeMaximum>40</parabolicDistributionSizeMaximum>
+                  </nearbySelection>
+              </valueSelector>
+          </changeMoveSelector>
+          <swapMoveSelector>
+              <entitySelector id="entitySelector2"/>
+              <secondaryEntitySelector>
+                  <nearbySelection>
+                      <originEntitySelector mimicSelectorRef="entitySelector2"/>
+                      <nearbyDistanceMeterClass>org.optaplanner.examples.tsp.domain.solver.nearby.VisitNearbyDistanceMeter</nearbyDistanceMeterClass>
+                      <parabolicDistributionSizeMaximum>40</parabolicDistributionSizeMaximum>
+                  </nearbySelection>
+              </secondaryEntitySelector>
+          </swapMoveSelector>
+          <tailChainSwapMoveSelector>
+              <entitySelector id="entitySelector3"/>
+              <valueSelector>
+                  <nearbySelection>
+                      <originEntitySelector mimicSelectorRef="entitySelector3"/>
+                      <nearbyDistanceMeterClass>org.optaplanner.examples.tsp.domain.solver.nearby.VisitNearbyDistanceMeter</nearbyDistanceMeterClass>
+                      <parabolicDistributionSizeMaximum>40</parabolicDistributionSizeMaximum>
+                  </nearbySelection>
+              </valueSelector>
+          </tailChainSwapMoveSelector>
+      </unionMoveSelector>
+    <acceptor>
+      <lateAcceptanceSize>200</lateAcceptanceSize>
+    </acceptor>
+    <forager>
+      <acceptedCountLimit>1</acceptedCountLimit>
+    </forager>
+  </localSearch>
+</solver>
```

Comment:
Please keep XML files formatted according to KIE Conventions, which requires 2 space indentation. Applies to data files too.